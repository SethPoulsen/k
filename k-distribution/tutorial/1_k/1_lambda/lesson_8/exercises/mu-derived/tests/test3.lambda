(
(lambda n . letrec timesN x = if x <= 1 then n else (n + (timesN (x + -1)))
        in (timesN 100))
5)