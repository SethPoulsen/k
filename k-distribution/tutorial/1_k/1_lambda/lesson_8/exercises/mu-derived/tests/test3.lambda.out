<k>
  500
</k>
